modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/AbstractEventEndpoint.java

```diff
@@ -2051,19 +2055,29 @@ public Response getNewConflicts(@FormParam("metadata") String metadata) throws N
 
   private List<JValue> convertToConflictObjects(final String eventId, final List<MediaPackage> events) throws SearchIndexException {
     final List<JValue> eventsJSON = new ArrayList<>();
-    for (final MediaPackage event : events) {
-      final Opt<Event> eventOpt = getIndexService().getEvent(event.getIdentifier().compact(), getIndex());
-      if (eventOpt.isSome()) {
-        final Event e = eventOpt.get();
-        if (StringUtils.isNotEmpty(eventId) && eventId.equals(e.getIdentifier())) {
-          continue;
+    final Organization organization = getSecurityService().getOrganization();
+    final User user = SecurityUtil.createSystemUser(systemUserName, organization);
+
+    SecurityUtil.runAs(getSecurityService(), organization, user, () -> {
+      try {
+        for (final MediaPackage event : events) {
+          final Opt<Event> eventOpt = getIndexService().getEvent(event.getIdentifier().compact(), getIndex());
+          if (eventOpt.isSome()) {
+            final Event e = eventOpt.get();
+            if (StringUtils.isNotEmpty(eventId) && eventId.equals(e.getIdentifier())) {
+              continue;
+            }
+            eventsJSON.add(convertEventToConflictingObject(e.getTechnicalStartTime(), e.getTechnicalEndTime(), e.getTitle()));
```

Comment:
Doesn't this leak the event titles?  If you don't have access to these events, shouldn't you not know their titles?

Reply:
Yes, it does return event title.